Mathlib/CategoryTheory/Limits/Types/Colimits.lean

```diff
@@ -326,6 +239,22 @@ theorem jointly_surjective' (x : colimit F) :
 /-- If a colimit is nonempty, also its index category is nonempty. -/
 theorem nonempty_of_nonempty_colimit {F : J ⥤ Type u} [HasColimit F] :
     Nonempty (colimit F) → Nonempty J :=
-  Nonempty.map <| Sigma.fst ∘ Quot.out ∘ (colimitEquivQuot F).toFun
+  Nonempty.map <| Sigma.fst ∘ Quot.out ∘ (colimitEquivColimitType F).toFun
+
+@[deprecated (since := "2025-03-27")] alias Quot.Rel := Functor.ColimitTypeRel
+@[deprecated (since := "2025-03-27")] alias Quot := Functor.ColimitType
+@[deprecated (since := "2025-03-27")] alias Quot.ι := Functor.ιColimitType
+@[deprecated (since := "2025-03-27")] alias Quot.jointly_surjective :=
+  Functor.ιColimitType_jointly_surjective
+@[deprecated (since := "2025-03-27")] alias Quot.desc := Functor.descColimitType
+@[deprecated (since := "2025-03-27")] alias Quot.ι_desc := Functor.descColimitType_comp_ι
+@[deprecated (since := "2025-03-27")] alias Quot.map_ι := Functor.ιColimitType_map
+@[deprecated (since := "2025-03-27")] alias isColimit_iff_bijective_desc :=
+  isColimit_iff_coconeTypesIsColimit
+@[deprecated (since := "2025-03-27")] alias colimitEquivQuot := colimitEquivColimitType
+@[deprecated (since := "2025-03-27")] alias colimitEquivQuot_symm_apply :=
+  colimitEquivColimitType_symm_apply
+@[deprecated (since := "2025-03-27")] alias colimitEquivQuot_apply :=
+  colimitEquivColimitType_apply
```

Comment:
We should perhaps bump the deprecation date so that the "6-month deprecation notice" gets respected? same with other deprecations in the PR?

Reply:
Thanks for the suggestion.